taxonomies/ingredients.txt

```diff
@@ -21184,8 +21335,17 @@ es:Saborizante de manzana natural
 fi:luontainen omena-aromi
 fr:arôme naturel de pomme
 hu:természetes alma aroma
+is:náttúruleg eplabragðefni
 nl:natuurlijk appelaroma
 
+<en:natural apple flavouring
+<en:with other natural flavouring
+en:natural apple flavouring with other natural flavourings
```

Comment:
we should remove this entry, and file a bug to improve the ingredients parser so that it can recognize it automatically. We do not want to create entries for all possible "natural [something]  flavour with other natural flavourings"

Reply:
Removed and filed: #4666